### PR TITLE
PD-1124: Updates uiColors.gray.base

### PR DESCRIPTION
**Pull request by brookescarlett**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [PD-1124](https://jira.mongodb.org/browse/PD-1124)

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: packages/palette/src/ui-colors.less [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
+@leafygreen__yellow--base: #ffdd49;
+@leafygreen__yellow--light-2: #fef2c8;
+@leafygreen__yellow--light-3: #fef7e3;
 
-@leafygreen__red--dark-3: #570B08; 
-@leafygreen__red--dark-2: #8F221B; 
-@leafygreen__red--base: #CF4A22; 
-@leafygreen__red--light-2: #F9D3C5; 
-@leafygreen__red--light-3: #FCEBE2; 
+@leafygreen__red--dark-3: #570b08;
```
Comment (DesignerDave) [inline packages/palette/src/ui-colors.less:28]: Why were these all lowercased?

Comment (brookescarlett) [reply packages/palette/src/ui-colors.less:28]: happened when i ran my formatter

Comment (DesignerDave) [reply packages/palette/src/ui-colors.less:28]: Wow that's awful, and I don't know why it hasn't formatted this file in the past...

We use all uppercase in JS. Is there a rule we can change?

Comment (brookescarlett) [reply packages/palette/src/ui-colors.less:28]: Looking into it

Comment (brookescarlett) [reply packages/palette/src/ui-colors.less]: Did it directly in Githubs UI since my editor was not having it

**Review by DesignerDave — APPROVED** [review]

**Pull request merged**